### PR TITLE
feat : added authentication

### PR DESCRIPTION
**Pull request by Digambar1411**
Authentication
added login and 
signup page

image
image

### CODE REVIEW

**Review by Rohitprasad83 — CHANGES_REQUESTED** [review]
Navbar is not visible, as per the figma screen the page should also contain navigation.

**Comment by Digambar1411** [comment]
Yes header I will add later

File: auth/auth.css [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+.login-card{
+    margin: 5rem auto;
+    min-width:280px;
+    width:310px;
+    background-color: var(--color-card);
+    padding:2rem;
+    border-radius: 4px;
+    height: fit-content; 
+    box-shadow: 0 4px 4px 0 rgb(0 0 0 / 20%);
```
Comment (vaibhav18matere) [inline auth/auth.css:29]: try to use common unit, either rem or px, I will suggest to use rem

File: auth/auth.css [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+    flex-direction: row;
+    gap: 2px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+.flex-left{
+    display: flex;
+    justify-content: flex-start;
+}
```
Comment (vaibhav18matere) [inline auth/auth.css:54]: you can make a separate class for common css properties or just use the properties made in component library

File: auth/login/login.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        <div class="flex-col">
+            <label class="larger-font" for="password">Password</label>
+            <input class="input padding-top-12px" placeholder="enter your password" id='password' type="password">
+        </div>
+
+        <div class="flex-col margin-top-8px">
+            <div class="flex-row">
+                <label class="flex-row">
+                    <input class="input-checkbox" type="checkbox" name="light"  value="">
+                    <span>Remember me</span>
```
Comment (vaibhav18matere) [inline auth/login/login.html:29]: try to provide values as well, it increases performance in screen reading

File: auth/signup/signup.html [hunk 4]
```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../auth.css">
+    <link href="https://fonts.googleapis.com/css2?family=Material+Icons+Outlined"
+      rel="stylesheet">
```
Comment (vaibhav18matere) [inline auth/signup/signup.html:9]: you can add all fonts in one common file and use whenever required

File: auth/signup/signup.html [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+            <div class="flex-left">
+                <label class="flex-row">
+                    <input class="input-checkbox" type="checkbox" name="light"  value="">
+                    <span>Accept terms & condition</span>
+                </label>
+                
+            </div>
+
+        </div>
+
```
Comment (vaibhav18matere) [inline auth/signup/signup.html:35]: remove blank lines, use prettier extension, it makes such small things easier & faster

**Review by vaibhav18matere — APPROVED** [review]

File: auth/auth.css (hunk 2 above)
Comment (Digambar1411) [reply auth/auth.css:54]: yeah i ll make use of it

**Pull request merged**